.github/workflows/ci.yml

```diff
@@ -26,6 +26,12 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install nextest test runner
         uses: taiki-e/install-action@nextest
+
+      - name: Build and run dev container task
+        uses: devcontainers/ci@v0.3
```

Comment:
hm can we use the prebuilt one? we might have to update it (and eventually automate it)

Reply:
sure, just trying it out for now